cmd/csi-addons/replication.go

```diff
@@ -81,6 +97,35 @@ type EnableVolumeReplication struct {
 	VolumeReplicationBase
 }
 
+func (v VolumeReplicationBase) setReplicationSource(req *proto.ReplicationSource) error {
+	if req == nil {
```

Comment:
use 
```
switch{
  case (condition):
  ..
   case (condition): 
}
```

format instead of continuous if blocks?

Reply:
+ follow order of

- req ==nil
- both volumeID and groupID set
- volumeID set
- groupID set
outside error out telling neither is set.

Like followed in Init() function ?